docs/tutorials/fundamentals/tutorial-swapping-token.md

```diff
@@ -0,0 +1,49 @@
+---
+title: 'Tutorial - Swapping UOS from Ultra to Ethereum'
+order: -99993
+outline: [0, 7]
+---
+
+# Tutorial - Swapping UOS from Ultra to Ethereum
+
+This tutorial guides you through the process of swapping UOS tokens from the Ultra network to UOS ERC-20 tokens on the Ethereum network using the Ultra Client. This process is designed for users who already have an Ethereum account and are familiar with cryptocurrency transactions.
+
+## Step 1: Prepare the Swap in the Ultra Client
+
+Start the swap process by opening the Ultra Client. Navigate to the Token Swap section.
+
+1. Enter the amount of UOS you want to swap. In this example, we will swap 3000 UOS.
+2. Enter your Ethereum address where you want to receive the UOS ERC-20 tokens. Make sure this is an address you control.
+
+![](./images/ultra-swap-ultra-client.png)
+
+Once you have entered the UOS amount and your Ethereum address, confirm the details and initiate the swap. This process will interact with the blockchain to transfer your UOS tokens and issue the equivalent amount in UOS ERC-20 on the Ethereum network.
+
+## Step 2: Verify the Swap on Ethereum
+
+After executing the swap transaction, you can verify your ERC-20 UOS tokens on the Ethereum network using Etherscan.
+
+1. Go to [https://etherscan.io](https://etherscan.io).
+2. Search for your Ethereum address to view the updated balance and incoming transactions.
+
+![](./images/ultra-swap-ether-scan.png)
+
+### Check Transaction Details
+
+To see the details of the transaction, including the amount of UOS ERC-20 received and the transaction hash, click on the specific transaction listed in your transaction history.
+
+![](./images/ultra-swap-ether-scan-txn-details.png)
+
+## Step 3: Verify the Transaction on the Ultra Network
```

Comment:
This is better done as a step 2 instead of 3, because the transaction on Ultra will happen first